apps/src/StudioApp.js

```diff
@@ -1811,6 +1811,25 @@ StudioApp.prototype.handleEditCode_ = function (config) {
   var aceEditor = this.editor.aceEditor;
   aceEditor.session.setMode('ace/mode/javascript_codeorg');
 
+  // Extend the command list on the ace Autocomplete object to include the period:
+  var Autocomplete = window.ace.require("ace/autocomplete").Autocomplete;
+  Autocomplete.prototype.commands['.'] = function(editor) {
```

Comment:
Ace docs are a little lacking - especially for something like autocomplete which is an extension. The source code for autocomplete is: `apps/lib/ace/src-noconflict/ext-language_tools.js`.

The goal here is to make sure that we preserve the autocomplete experience for `Math.min` and `console.log` (don't dismiss when the period is typed), but normally make it dismiss at the period and allow a new method/property specific completer to appear when the first character is typed after the period.